phoenix-core/src/test/java/org/apache/phoenix/schema/MetaDataClientTest.java

```diff
@@ -7,48 +7,45 @@
 import org.apache.phoenix.parse.PSchema;
 import org.apache.phoenix.parse.SQLParser;
 import org.apache.phoenix.query.BaseConnectionlessQueryTest;
-import org.apache.phoenix.schema.types.PDataType;
 import org.apache.phoenix.util.EnvironmentEdgeManager;
 import org.junit.Test;
-import org.mockito.Mockito;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
 import java.sql.DriverManager;
 import java.sql.SQLException;
-import java.util.HashMap;
-import static junit.framework.Assert.fail;
 import static junit.framework.TestCase.assertEquals;
-import static org.mockito.Matchers.any;
+import static junit.framework.TestCase.assertTrue;
+import static junit.framework.TestCase.fail;
 
 public class MetaDataClientTest extends BaseConnectionlessQueryTest {
-    private static final Logger LOGGER = LoggerFactory.getLogger(MetaDataClientTest.class);
 
     @Test
-    public void testCreateTable() throws SQLException {
+    public void TestHandleCreateTableMutationCode() throws SQLException {
```

Comment:
I would break this test into two parts, may consider further refactoring to check the exception code passed in as a parameter to a function used for both the tests.